src/man/sssd-ldap.5.xml

```diff
@@ -1778,7 +1778,17 @@
                             principal used for authentication to the directory.
                             This option can either contain the full principal (for
                             example host/myhost@EXAMPLE.COM) or just the principal name
-                            (for example host/myhost).
+			    (for example host/myhost). By default, the value is not set
+			    and the following principals are used. If none of them are
+			    found, the first principal in keytab is returned.
+<programlisting>
+- hostname@REALM
+- netbiosname$@REALM
+- host/hostname@REALM
+- *$@REALM
+- host/*@REALM
+- host/*
```

Comment:
For those, I'm not sure if we want the "- " in the beginning of the line.
Also, IMO, the open and close <programlisting> tags should be one level inside the <para> tags.

Reply:
+1